src/driver/communication_interface.cc

```diff
@@ -554,10 +577,19 @@ void CommunicationInterface::HandlePlan(
     // Cartesian goal command. Not implemented yet. Ignore for now
   }
   lock.unlock();
+
+  auto t_end {hr_clock::now()};
```

Comment:
A few nits that don't matter:
- These local variables can go straight into your `timepoints_` object
- The `t1 t2 t3` in communication_interface.cc could use better names.
But I'm approving with or without updates.